docs/examples/logs/README.rst

```diff
@@ -0,0 +1,75 @@
+OpenTelemetry Logs SDK
+======================
+
+Start the Collector locally to see data being exported. Write the following file:
+
+.. code-block:: yaml
+
+    # otel-collector-config.yaml
+    receivers:
+        otlp:
+            protocols:
+                grpc:
+  
+    exporters:
+        logging:
```

Comment:
I think this needs to be set to `loglevel: debug` if we want the log output shown below.

Reply:
Below example script has logs with different severities including error which is always shown I guess. I tested this manually before committing. I didn't have to set this log level.